pkgs/test_core/lib/src/runner/engine.dart

```diff
@@ -174,22 +169,22 @@ class Engine {
   Set<LiveTest> get failed => _failedGroup.set;
   final _failedGroup = UnionSetController<LiveTest>(disjoint: true);
 
-  /// The tests that are still running, in the order they begain running.
+  /// The tests that are still running, in the order they began running.
   List<LiveTest> get active => UnmodifiableListView(_active);
   final _active = QueueList<LiveTest>();
 
+  /// The tests from [LoadSuite]s that are still running, in the order they
+  /// began running.
```

Comment:
Might as well change the wording here to stop referring to them as "tests" as well.

```suggestion
  /// The [LoadSuite]s that are still loading, in the order they began.
```

Reply:
Done.